src/scry/completion/dependency_graph.cr

```diff
@@ -28,12 +28,9 @@ module Scry::Completion::DependencyGraph
     getter prelude_node : Node
 
     def initialize(@nodes = {} of String => Node)
-      if crystal_path = ENV["CRYSTAL_PATH"]?.to_s.split(":").last?
-        prelude_path = "#{crystal_path}/prelude.cr"
-        @nodes[prelude_path] = @prelude_node = Node.new(prelude_path)
-      else
-        @prelude_node = Node.new("")
-      end
+      crystal_path = Crystal::DEFAULT_PATH.split(":").last
+      prelude_path = "#{crystal_path}/prelude.cr"
```

Comment:
I think you should be able to use `Crystal::CrystalPath.new.find "prelude.cr"`

Reply:
Oh,nice, let me try :+1:

Reply:
@bew Seems this requires the entire crystal compiler, even LLVM :sweat_smile: 

```
Error in src/scry.cr:8: instantiating 'Crystal::CrystalPath:Class#new()'

p Crystal::CrystalPath.new.find "prelude.cr"
                       ^~~

in /usr/lib/crystal/compiler/crystal/crystal_path.cr:14: instantiating 'Crystal::Config:Module#default_target_triple()'

    def initialize(path = CrystalPath.default_path, target_triple = Crystal::Config.default_target_triple)
                                                                                    ^~~~~~~~~~~~~~~~~~~~~

in /usr/lib/crystal/compiler/crystal/config.cr:61: undefined constant LLVM

      {{env("CRYSTAL_CONFIG_TARGET")}} || LLVM.default_target_triple
```

Reply:
@bew Well, I already did that [here](https://github.com/crystal-lang-tools/scry/blob/8653d6367ab2e1c8eb1d1a6e6fe16c00fa891e18/src/scry/missing_methods.cr#L1), although, `CrystalPath` seems to require way more code :sweat_smile:

Reply:
...and LLVM :no_good_man:

Reply:
Oh, I get the idea :+1: Would be nice to open a new issue to track and fix this then :wink:

Reply:
Ok, I just tried to copy compiler code from [here](https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/crystal_path.cr)... I just got too many errors :sweat_smile: , I guess this can be achieved in the future, for now I think using `ENV["CRYSTAL_PATH"]` is a nice approach :+1:

Reply:
Also, this is just to find `prelude.cr`, this file is always on the root of `CRYSTAL_PATH` on macOS and Linux (already tested in both by TravisCI :tada: ). If someone needs other prelude, then scry isn't capable to do that yet, since the [command here](https://github.com/crystal-lang-tools/scry/blob/master/src/scry/analyzer.cr#L55) should be modified.